src/celengine/overlay.h

```diff
@@ -77,14 +79,15 @@ class Overlay
 
     float xoffset{ 0.0f };
     float yoffset{ 0.0f };
+    TextureFont::Alignment textAlignment{ TextureFont::Alignment::Left };
 
     Renderer& renderer;
 
     struct CursorPosition
     {
         void reset()
         {
-            x = y = 0.125f;
```

Comment:
this was set intentionally to prevent jittering. i don't understand completely how it works but it was in 1.6 and with positions set to 0 we had labels jittering.

Reply:
I only found a note here https://github.com/CelestiaProject/Celestia/blob/b0f40cec084d85ce767e35d9f2d344fe12b72be1/src/celengine/render.cpp#L108 (which I also deleted). I did not see anything abnormal removing this.